app/api/custom/orders.py

```diff
@@ -93,3 +102,72 @@ def calculate_amount():
             return UnprocessableEntityError({'source': 'discount-code'}, 'Discount Usage Exceeded').respond()
 
     return jsonify(calculate_order_amount(tickets, discount_code))
+
+
+@order_blueprint.route('/create-order', methods=['POST'])
+@jwt_required
+def create_order():
+    data = request.get_json()
+    tickets = data['tickets']
+    attendee = data['attendee']
+    for attribute in attendee:
+        attendee[attribute.replace('-', '_')] = attendee.pop(attribute)
+    schema = AttendeeSchema()
+    json_api_attendee = {"data": {"attributes": data['attendee'], "type": "attendee"}}
+    result = schema.load(json_api_attendee)
+    if result.errors:
+        return jsonify(result.errors)
+    ticket_ids = [int(ticket['id']) for ticket in tickets]
+    ticket_info_list = db.session.query(Ticket).filter(Ticket.id.in_(ticket_ids)).filter_by(event_id=data['event_id'],
+                                                                                            deleted_at=None).all()
+    ticket_ids_found = [ticket_information.id for ticket_information in ticket_info_list]
+    tickets_not_found = list(set(ticket_ids) - set(ticket_ids_found))
+    if tickets_not_found:
+        return jsonify(status='Order Unsuccessful', error='Ticket with id {} was not found.'.format(tickets_not_found))
```

Comment:
This will send HTTP status code as 200, which means OK. Which means order was created successfully